osu.Game/Screens/Menu/IntroScreen.cs

```diff
@@ -109,7 +110,9 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
 
         protected void PrepareMenuLoad()
         {
-            LoadComponentAsync(mainMenu = new MainMenu());
```

Comment:
What's the difference?

Reply:
Any big differences, but I think that it would be better for read.